presto-parser/src/test/java/com/facebook/presto/sql/parser/TestSqlParser.java

```diff
@@ -150,6 +150,23 @@ public void testPossibleExponentialBacktracking()
     }
 
     @Test
+    public void testQualifiedName()
+    {
+        QualifiedName name = QualifiedName.of("a", "b", "c", "d");
```

Comment:
I'd just inline all these variables. They are simple assertions, and it would be easier to read if they read like:

```java
assertTrue(QualifiedName.of("a", "b", "c", "d").hasSuffix(QualifiedName.of("c", "d"));
assertFalse(QualifiedName.of("a", "b", "c", "d").hasSuffix(QualifiedName.of("a", "b"));
...
```